src/utils/editor/source-search.js

```diff
@@ -238,16 +257,23 @@ function findNext(
  */
 function findPrev(
   ctx: any, query: string, keepSelection :boolean, modifiers: SearchModifiers) {
-  doSearch(ctx, true, query, keepSelection, modifiers);
+  return doSearch(ctx, true, query, keepSelection, modifiers);
 }
 
 function countMatches(
-  query: string, text: string, modifiers: SearchModifiers): number {
-  const regexQuery = buildQuery(query, modifiers, {
-    isGlobal: true
-  });
-  const match = text.match(regexQuery);
-  return match ? match.length : 0;
+  ctx: any, query: string, text: string, modifiers: SearchModifiers): number {
```

Comment:
@wldcordeiro, Thanks! I think `countMatches` do not need `text` as a parameter, but I'm not very sure, so I have not remove it.